examples/Test/Automated/006_CA_ORU_R01_LOINC.hl7

```diff
@@ -0,0 +1,7 @@
+MSH|^~\&|SISGDSP|SISGDSP|SISHIERECEIVER^11903029^L,M,N|^automated-staging-test-receiver-id^L,M,N|20240226034304||ORU^R01^ORU_R01|006|N|2.5.1
```

Comment:
I tried running the assertions on these files but for some reason the Hapi library is not able to parse them. It triggers an error when trying to parse the 2.5.1 version  number: `ca.uhn.hl7v2.HL7Exception: The HL7 version 2.5.1\nPID is not recognized`. I also noticed the files Gilmore added are not getting parsed either. Could you try adding a `|` after `2.5.1`? That may fix it

Reply:
I updated the 4 files. Could you try them again? *Also, what's your process for running the assertions?

Reply:
I run the `Automated Staging Test - Submit Messages` github workflow on the branch to send the new files to RS staging. You can then check they are delivered to the RS blob storage account `cdctiautomatedstg` (you need to log in to your -SU account to be able to check this). After making sure they are in the storage account, you can then run and debug the `AutomatedTest` test in TI. To be able to run it, you need to set the `AZURE_STORAGE_CONNECTION_STRING` for the `cdctiautomatedstg` account. I can give that to you. Otherwise, you can also run the `Automated Staging Test - Run integration tests` workflow, but you won't be able to debug there